Cargo.toml

```diff
@@ -65,6 +65,13 @@ kona-protocol = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1
 kona-registry = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.14", default-features = false }
 kona-genesis = { git = "https://github.com/op-rs/kona", tag = "kona-client/v0.1.0-beta.14", default-features = false }
 
+# hana
+# TODO: use tag when https://github.com/celestiaorg/hana/pull/5 is merged and new release is cut.
```

Comment:
TODO

Reply:
I'll ask diego to cut a new release and update it before merging, when I'm done with some e2e testing and making sure it works.

Currently the nodes have rate limits, so I'm getting throttled which is blocking me to run e2e tests.